contracts/protocol/facets/SellerHandlerFacet.sol

```diff
@@ -84,6 +86,8 @@ contract SellerHandlerFacet is SellerBase {
             ADMIN_OR_AUTH_TOKEN
         );
 
```

Comment:
Should we not ensure that the check happens when the seller.admin == address(0) and when the AuthTokenType is not AuthTokenType.None ?

Reply:
Ok, apologies this does make sense to me. 

At this point in the code the address is either address(0) and NOT AuthTokenType.None OR is it NOT address(0) and AuthTokenType.None... 

which implies that if at this line, if it is NOT Null, it will be rejected if .Custom ...